src/main/host/descriptor/mod.rs

```diff
@@ -116,6 +116,8 @@ bitflags::bitflags! {
         const WRITABLE = c::_Status_STATUS_FILE_WRITABLE;
         /// User already called close.
         const CLOSED = c::_Status_STATUS_FILE_CLOSED;
+        /// Flipped when the size of the input buffer changes
+        const INPUT_BUFFER_PARITY = c::_Status_STATUS_FILE_INPUT_BUFFER_PARITY;
```

Comment:
Yeah, I understand that this is quite hacky. Since I think that this feature probably requires big architectural change and we don't know yet what it should be, I implemented the easiest and shortest possible approach.